deploy/util/fsspec_utils.py

```diff
@@ -0,0 +1,27 @@
+import logging
+from os import PathLike, fspath
+from fsspec.core import url_to_fs # type: ignore
+from pathlib import Path
+
+rootLogger = logging.getLogger()
+
+
+def downloadURI(uri: str, local_dest_path: PathLike):
```

Comment:
```suggestion
def downloadURI(uri: str, local_dest_path: PathLike) -> None:
```
We've gotten in the habit of being very through with the type hints...